common-arcana.lic

```diff
@@ -87,6 +91,12 @@ module DRCA
     case DRC.bput(cast_command || 'cast', get_data('spells').cast_messages)
     when /^Your target pattern dissipates/, /^You can't cast that at yourself/, /^You need to specify a body part to consume/
       fput('release spell')
+    when /^At the ritual's peak, your prophetic connection blooms a thousand-fold/
+      if Flags['rtr-expire'].nil? 
+        Flags.add('rtr-expire', get_data('spells').spell_data['Read the Ripples']['expire'])
```

Comment:
I think it might be easier to add the Flag in `astrology`, then you can reset it here instead of conditionally adding it.